dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/controller/TempTestController.java

```diff
@@ -111,6 +131,59 @@ public String forwardConcorContribution(@RequestBody ConcorContributionReqForDrc
         return "OK";
     }
 
+    @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE))
+    @PostMapping(value = "/send-contributions")
+    @Operation(description = "<h3> ** Use with caution, only if you are certain !!! ** </h3>"
+        + "*This operation is not part of normal processing*<br>"
+        + "*It bypasses normal processing and can send production data to a third party*<br>"
+        + "<ul>"
+        + "<li>Obtains XML for each concor contribution ID in the list.</li>"
+        + "<li>Sends it to the DRC</li>"
+        + "<li>Does NOT create a Contributions File for the sent concor contributions.</li>"
+        + "<li>Does NOT update each successfully processed concor contribution to SENT in MAAT.</li>"
+        + "</ul>"
+        + "*Only to be used during testing or fault-finding*<br><br>"
+        + "When given a list of Concor Contribution IDs, get the XML for each and send them to the DRC"
+        + "Returns the list of Concor Contribution Entries (ID + XML) that were sent to DRC")
+
+    public ResponseEntity<List<ConcorContribEntry>> sendConcorContributionXmlsToDRC(@RequestBody List<Long> idList) {
+
+        log.info("Request received to get and send the XML for {} IDs", idList.size());
+        if (idList.isEmpty()) {
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "ID List Empty");
+        } else if (idList.size() > REQUEST_ID_LIST_SIZE_LIMIT_CONCOR) {
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Too many IDs provided, max is " + REQUEST_ID_LIST_SIZE_LIMIT_CONCOR);
+        } else {
+            return ResponseEntity.ok(concorContributionsService.sendContributionsToDrc(idList));
+        }
+    }
+
+    @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE))
+    @PostMapping(value = "/send-fdcs")
+    @Operation(description = "<h3> ** Use with caution, only if you are certain !!! ** </h3>"
+        + "*This operation is not part of normal processing*<br>"
+        + "*It bypasses normal processing and can send production data to a third party*<br>"
+        + "<ul>"
+        + "<li>Obtains XML for each FDC contribution ID in the list.</li>"
+        + "<li>Sends it to the DRC</li>"
+        + "<li>Does NOT create a Contributions File for the sent FDC contributions.</li>"
+        + "<li>Does NOT update each successfully processed FDC contribution to SENT in MAAT.</li>"
+        + "</ul>"
+        + "*Only to be used during testing or fault-finding*<br><br>"
+        + "When given a list of FDC Contribution IDs, get the XML for each and send them to the DRC"
+        + "Returns the list of FDC XMLs that were sent to DRC")
+    public ResponseEntity<List<Fdc>> sendFdcContributionXmlsToDRC(@RequestBody List<Long> idList) {
```

Comment:
Very minor thing, but I don't think this method uses XML at all (despite the method name and the `@Operation` description). If not, it might be worth changing the method name(s?) and `@Operation` description(s?) to avoid confusion.

Reply:
That's correct, now changed.